.changeset/heavy-comics-sell.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+fix: Fail intentionally on older versions of node.
```

Comment:
From the point of view of a changelog reader, I think this would be more understandable if it read something like:

```suggestion
fix: ensure that a helpful error message is shown when on unsupported versions of Node.js.
```

Reply:
I like this, thanks